Jenkinsfile

```diff
@@ -11,9 +11,9 @@ if (master_branches.contains(env.BRANCH_NAME)) {
     ])
 }
 
-task_wrapper('mesos-sec', master_branches){
+task_wrapper('mesos-sec', master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#dcos-security-ci') {
```

Comment:
@karlkfi is it possible to add a comment here to explain where we got the token `8b793652-f26a-422f-a9ba-0d1e47eb9d89` from?

Reply:
@asridharan It is described in the `sec-ci-libs` documentation: https://github.com/mesosphere/sec-ci-libs#standardised-task-wrapper

Reply:
Seems like it's a token passed to jenkins so that jenkins knows which credentials to use from its secret store. But it's not very maintenance-friendly as a magic string....
It should probably be either a string variable ro an environment variable injected by jenkins.